src/Sulu/Bundle/WebsiteBundle/Controller/ExceptionController.php

```diff
@@ -69,10 +70,12 @@ public function __construct(
 
     /**
      * {@see BaseExceptionController::showAction()}.
+     *
+     * @param LegacyFlattenException|FlattenException $exception
      */
     public function showAction(
         Request $request,
-        FlattenException $exception,
+        $exception,
```

Comment:
Can be a Symfony\Component\ErrorRenderer\Exception\FlattenException or the old FlattenExcpetion and not sure if my fix will be merged: https://github.com/symfony/symfony/pull/33918

Reply:
Moved to #4821.